applications/plugins/SofaPython/Binding_BaseObject.cpp

```diff
@@ -270,6 +276,7 @@ SP_CLASS_METHOD_DOC(BaseObject, getCategories,
 SP_CLASS_METHOD_DOC(BaseObject, getTarget,
                     "Returns the target (plugin) that contains the current object.")
 SP_CLASS_METHOD(BaseObject,getAsACreateObjectParameter)
+SP_CLASS_METHOD(BaseObject,computeBBox)
```

Comment:
It is better to use SP_CLASS_METHOD_DOC() to add a docstring so we can query the object from interactive python environemnt (like Ipython notebook) and we can generate the doc out of code.